### PR TITLE
Add emptyMessage prop to show when there is no data in Table

### PR DESCRIPTION
**Pull request by kennyt**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @rcorral

**Comment by rcorral** [comment]
Can we add a test or two?

**Comment by kennyt** [comment]
alright, done

File: src/Table/__tests__/Table-test.js [hunk 1]
```diff
@@ -49,4 +49,32 @@ describe('Table', function () {
...
+          columns={MockTable.columns}
+          data={[]}
+          emptyMessage={this.emptyMessage}
+          idAttribute={this.idAttribute}
+          sortBy={this.sortBy}
+          onSortCallback={this.callback} />
+      );
+    });
+
+    it('should display the custom empty message', function () {
```
Comment (rcorral) [inline src/Table/__tests__/Table-test.js]: Can we also have a test case for the inverse? To make sure that `'No data'` is used?

Comment (kennyt) [inline src/Table/__tests__/Table-test.js]: alright, done

**Pull request merged**